src/geojs-scene-manager/index.js

```diff
@@ -94,75 +89,52 @@ export class GeoJSSceneManager {
         fillColor: 'gray',
         strokeOpacity: 0.8,
         fillOpacity: 0.8,
-        radius: name => Math.max(2, Math.sqrt(nodes[name].degree)),
+        radius: 10,
       },
       position: name => nodes[name],
     })
       .data(Object.keys(nodes));
 
-    const ui = map.createLayer('ui', {
-      zIndex: 2,
-    });
-
-    const tooltip = ui.createWidget('dom', {
-      position: {
-        x: 0,
-        y: 0,
-      },
-    });
-
-    const tooltipElem = tooltip.canvas();
-    tooltipElem.setAttribute('id', 'tooltip');
-    tooltipElem.classList.toggle('hidden', true);
-    tooltipElem.style['pointer-events'] = 'none';
-
-    let onNode = false;
+    let onNode = null;
 
     points.geoOn(geo.event.feature.mouseon, evt => {
-      onNode = true;
+      onNode = this.dp.nodes[evt.data];
 
-      const name = evt.data;
-
-      tooltip.position(evt.mouse.geo);
-      tooltipElem.innerText = name;
-      tooltipElem.classList.toggle('hidden', false);
-    });
-    points.geoOn(geo.event.feature.mousemove, evt => {
-      tooltip.position(evt.mouse.geo);
+      // compute point dimensions
+      const radius = points.style("radius")(evt.data);
+      // position relative to canvas
+      const position = this.points.featureGcsToDisplay(points.position()(evt.data));
+      this.hovered(onNode, position, radius);
     });
     points.geoOn(geo.event.feature.mouseoff, evt => {
-      onNode = false;
-
-      tooltipElem.classList.toggle('hidden', true);
+      onNode = null;
+      this.hovered(null, null, null);
     });
     points.geoOn(geo.event.feature.mouseclick, evt => {
       if (evt.top) {
-        const data = this.pickName(evt.data);
-        this.picked(data, evt.mouse.map);
-      }
-    });
-
-    lines.geoOn(geo.event.feature.mouseon, evt => {
-      if (onNode) {
-        return;
+        this.picked(this.dp.nodes[evt.data], evt.mouse.geo);
       }
-
-      const text = `${evt.data[0]} - ${evt.data[1]}`;
-
-      tooltip.position(evt.mouse.geo);
-      tooltipElem.innerText = text;
-      tooltipElem.classList.toggle('hidden', false);
-    });
-    lines.geoOn(geo.event.feature.mousemove, evt => {
-      tooltip.position(evt.mouse.geo);
     });
-    lines.geoOn(geo.event.feature.mouseoff, evt => {
-      if (onNode) {
-        return;
-      }
 
-      tooltipElem.classList.toggle('hidden', true);
-    });
+    // NOTE: disable line hovering for now till figured out where to show
```

Comment:
Can you file an issue to describe what needs to be done here? Or, if you've created the issue already, could you just link it here?

Reply:
see https://github.com/arclamp/materialnet/issues/87